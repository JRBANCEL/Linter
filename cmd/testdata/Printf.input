package test

import (
	"log"
)

func PrintfTwoArgsSpace() {
	arg := "test"
	log.Printf("YOLO: %v", arg)
}

func PrintfTwoArgsNoSpace() {
	arg := "test"
	log.Printf("YOLO:%v", arg)
}

func PrintfThreeArgs() {
	arg0 := "test"
	arg1 := "yolo"
	log.Printf("YOLO %v and %v", arg0, arg1)
}