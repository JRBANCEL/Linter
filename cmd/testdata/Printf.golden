package test

import (
	"log"
)

func PrintfTwoArgsSpace() {
	arg := "test"
	log.Print("YOLO: ", arg)
}

func PrintfTwoArgsNoSpace() {
	arg := "test"
	log.Print("YOLO: ", arg)
}

func PrintfThreeArgs() {
	arg0 := "test"
	arg1 := "yolo"
	log.Printf("YOLO %v and %v", arg0, arg1)
}